Makefile

```diff
@@ -97,7 +97,7 @@ lint: deps
 	# check terraform fmt
 	$(TERRAFORM) fmt -check=true -write=false -diff=true ci/infra
 	# run golangci-lint
-	$(BINPATH)/golangci-lint run
+	$(BINPATH)/golangci-lint -v run --timeout 3m
```

Comment:
previous job timed out  in 1 min, this has to be something with the slave, it takes only a few seconds in my laptop and in this new run.